dynamiq/nodes/agents/react.py

```diff
@@ -1,17 +1,24 @@
 import json
 import re
+from enum import Enum
 from typing import Any
 
 from pydantic import Field
 
 from dynamiq.nodes.agents.base import Agent, AgentIntermediateStep, AgentIntermediateStepModelObservation
-from dynamiq.nodes.agents.exceptions import ActionParsingException, AgentMaxLoopsReached, RecoverableAgentException
+from dynamiq.nodes.agents.exceptions import ActionParsingException, MaxLoopsExceededException, RecoverableAgentException
 from dynamiq.nodes.node import NodeDependency
 from dynamiq.nodes.types import InferenceMode
 from dynamiq.prompts import Message, Prompt
 from dynamiq.runnables import RunnableConfig, RunnableStatus
 from dynamiq.utils.logger import logger
 
+
+class BehaviorOnMaxLoops(str, Enum):
```

Comment:
@acoola Do we need to replicate this enum in multiple places? Or should we reuse the same that we already have?

Such things might be common; at least we already have it in multiple places. So, I would probably put it somewhere in the `node` package and use it from there.

Reply:
Of course, we should limit duplication, so let's use some common name and move to `nodes/types.py`